### PR TITLE
Network host reachable check and other minor code refactoring

### PR DESCRIPTION
**Pull request by ajabble**
*Fixed items*

1) Unsupported URL and service host not found error handling added to N/w calls.
2) Prefix Header file fixes
3) Constants added to file instead of macro definition
4) Forgot Password view minor fixes.

### CODE REVIEW

File: AJOAuth2/Helper.h [hunk 1]
```diff
@@ -13,9 +13,10 @@
 
 + (BOOL)isConnected;
 + (BOOL)validateEmail:(NSString *)candidate;
-+ (void)userInfoSaveInDefaults:(NSDictionary *)userInfoDict;
 + (BOOL)checkResponseObject:(id)responseObject;
++ (BOOL)isWebUrlValid:(NSError *)error;
 + (User *)getUserPrefs;
++ (void)userInfoSaveInDefaults:(NSDictionary *)userInfoDict;
```
Comment (praveen-garg) [inline AJOAuth2/Helper.h]: userInfoSaveInDefaults > saveUserInfoInDefaults
userInfoDict > userInfo

File: AJOAuth2/Helper.m [hunk 2]
```diff
@@ -48,7 +51,14 @@ + (BOOL)checkResponseObject:(id)responseObject {
...
++ (BOOL)isWebUrlValid:(NSError *)error {
+    if (error.code == kErrorUnsupportedURL || error.code == kErrorCannotFindHost) {
+        [SVProgressHUD showErrorWithStatus:error.localizedDescription];
+        return NO;
+    }
+    
+    return YES;
+}
 + (User *)getUserPrefs {
     NSData *myObject = [PREFS objectForKey:USER_INFO];
```
Comment (praveen-garg) [inline AJOAuth2/Helper.m]: userInfoObject

**Review by praveen-garg — CHANGES_REQUESTED** [review]
reviewed

**Comment by ajabble** [comment]
@praveen-garg changes made as per feedback.

**Review by praveen-garg — APPROVED** [review]
looks good; (DRY run review)

**Pull request merged**